doc/classes/Transform2D.xml

```diff
@@ -123,6 +123,12 @@
 				Returns the inverse of the transform, under the assumption that the transformation is composed of rotation and translation (no scaling, use [method affine_inverse] for transforms with scaling).
 			</description>
 		</method>
+		<method name="is_conformal" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns true if the transform's basis is conformal, meaning it preserves angles and distance ratios, and may only composed of rotation or uniform scale. Returns false if the transform's basis has non-uniform scale or shear/skew. This can be used to validate if the transform is non-distorted, which is important for physics and other use cases.
```

Comment:
typo: "may only composed" -> "may only be composed"

Reply:
Might be confusing to use "or" here for people who use exclusive or rather than inclusive in their speech, "and" might be less confusing, and won't be wrong as `(1, 1)` is uniform